### DISCUSSION TITLE
Sconto Cassa sul Totale a Pagare

### DISCUSSION THREAD

**Discussion by MrLeonida** [fattureincloud/api#299] [Q&A] [answered]
Ciao,

In Fatture in Cloud, lato interfaccia Web, è possibile indicare uno sconto cassa sul totale a pagare nella sezione "Opzioni avanzate" > "Applica Sconto o Maggiorazione sul totale a pagare".

Utilizzando le API, quale campo va popolato con quel valore?
Se ho 105€ totale documento, basta indicare i 5€ (esempio) di sconto nel campo sopra e i 100€ di totale pagamenti?

Grazie.

**Comment by emajo** [comment]
Ciao @MrLeonida a quale tipo di documento si sta riferendo?

**Reply by MrLeonida** [reply]
Ciao! Stiamo provando a generare una Fattura di Vendita con le caratteristiche di cui sopra. Grazie.

**Comment by emajo** [comment, marked as answer]
Ciao @MrLeonida, per gestire sconto/maggiorazione sul totale il campo relativo è `amount_due_discount`, che può essere valorizzato con valori positivi o negativi a seconda della necessità.

Grazie,
Emanuele.

**Reply by MrLeonida** [reply]
Grazie, nel mio caso, essendo uno sconto, andava valorizzato con segno negativo.
